src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrent.cs

```diff
@@ -360,10 +360,30 @@ public override DownloadClientItem GetImportItem(DownloadClientItem item, Downlo
 
         public override DownloadClientInfo GetStatus()
         {
+            var version = Proxy.GetApiVersion(Settings);
             var config = Proxy.GetConfig(Settings);
 
             var destDir = new OsPath(config.SavePath);
 
+            if (Settings.TvCategory.IsNotNullOrWhiteSpace() && version >= Version.Parse("2.0"))
+            {
+                var label = Proxy.GetLabels(Settings)[Settings.TvCategory];
+
+                if (label.SavePath.IsNotNullOrWhiteSpace())
```

Comment:
With the above change:

```suggestion
                if (label?.SavePath.IsNotNullOrWhiteSpace())
```